dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides

```diff
@@ -0,0 +1,25 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2
+
+Bundle-Activator: com.ibm.ws.cxf.rt.bindings.soap.NoOpActivator
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+exportVer=3.2.6
+
+Export-Package: \
+  com.ibm.ws.cxf.rt.bindings.soap
+
+Import-Package: \
```

Comment:
Are we overriding this to not import any packages?  The bundle would never resolve like this.  It would be better to just leave the artifact's default import statement in place by removing this line

Reply:
I'm good with defaults as long as it doesn't break a build. I don't know what packages we need to import/export at this point.